### PR TITLE
APM Logs-in-Context Forwarding on by default

### LINKED ISSUE #1037: Enabled Logging by Default
Modify our default configuration to enable logging by default.

### PR DESCRIPTION
**Pull request by jaffinito**
## Description

APM Logs-in-Context Forwarding on by default in XSD.  Will turn on for anyone who upgrades that does not explicitly disable it.

Closes #1037 

# Author Checklist
- [x] Unit tests, Integration tests, and Unbounded tests completed
- [x] Performance testing completed with satisfactory results (if required)
- [x] [Agent Changelog](/src/Agent/CHANGELOG.md) or [Lambda Agent changelog](/src/AwsLambda/CHANGELOG.md) updated 

# Reviewer Checklist
- [ ] Perform code review
- [ ] Pull request was adequately tested (new/existing tests, performance tests)
- [ ] Review Changelog

### CODE REVIEW

**Review by vuqtran88 — CHANGES_REQUESTED** [review]
Do we need to update the integration tests to not explicitly enable forwarding ?

**Comment by jaffinito** [comment]
No, I think it is good to be explicit in the tests so that it is clear when reading them what is or is not enabled.

**Review by JcolemanNR — CHANGES_REQUESTED** [review]
Needs to have the code regenerated from xsd to incorporate the new default value! :)

**Comment by vuqtran88** [comment]
Just my 2 cents. I think there is a benefit of not explicitly enabling forwarding. The benefit is we can actually test the enable by default behavior. We actually did that for other enabled-by-default  configurations (.ie DT, headers capture ...). For readability, i think a comment in the test would be efficient.

**Comment by JcolemanNR** [comment]
A unit test needs to be updated or created to test the default configuration as well

**Comment by jaffinito** [comment]
@JcolemanNR Nice catch!

File: src/Agent/NewRelic/Agent/Core/Config/Configuration.cs [hunk 1]
```diff
@@ -1,6 +1,3 @@
-// Copyright 2020 New Relic, Inc. All rights reserved.
-// SPDX-License-Identifier: Apache-2.0
-
```
Comment (JcolemanNR) [inline src/Agent/NewRelic/Agent/Core/Config/Configuration.cs:3]: Alex and I have been manually adding this copyright back in... not sure if it matters though

**Review by JcolemanNR — APPROVED** [review]
Looks good! Minor comment about copyright, but I don't know if that's really needed on generated files

File: src/Agent/NewRelic/Agent/Core/Config/Configuration.cs (hunk 1 above)
Comment (jaffinito) [reply src/Agent/NewRelic/Agent/Core/Config/Configuration.cs]: Fixed!

**Pull request merged**